src/features/world/ui/trader/BuyPanel.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useContext, useState } from "react";
+import { useActor } from "@xstate/react";
+
+import { Context } from "features/game/GameProvider";
+import { ITEM_DETAILS } from "features/game/types/images";
+
+import { Button } from "components/ui/Button";
+
+import { useAppTranslation } from "lib/i18n/useAppTranslations";
+import { OuterPanel } from "components/ui/Panel";
+import { Box } from "components/ui/Box";
+import Decimal from "decimal.js-light";
+import token from "assets/icons/token_2.png";
+import { TRADE_LIMITS } from "features/game/events/landExpansion/listTrade";
+import { getKeys } from "features/game/types/craftables";
+import { InventoryItemName } from "features/game/types/game";
+import { SUNNYSIDE } from "assets/sunnyside";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import {
+  Listing,
+  getTradeListings,
+} from "features/game/actions/getTradeListings";
+import { hasFeatureAccess } from "lib/flags";
+
+export const BuyPanel: React.FC = () => {
```

Comment:
Do we need to add Gold Pass protection on this route?

When you sell it is correctly disabling the feature (without a Gold Pass), but it still let's a player search for items.

At what point do you think they should be blocked?